_src/assets/js/addcolors.js

```diff
@@ -27,76 +27,83 @@ function handleThirdPalete() {
 
 function addFirstPalete() {
```

Comment:
Hi @rdfornieles I think you could create one function for Palletes since the addFirstPalete, addSecondPalete and addThirdPalete are the same. This way you can use parameters when you call to the functions. For example you could create a funcion addPalete (parameter,paramenter...) when you need the firts palete call the function: addPalete (firspalete parameter) and the same for the second and the third palete